PWGHF/TableProducer/candidateSelectorDplusToPiKPi.cxx

```diff
@@ -230,9 +226,9 @@ struct HfCandidateSelectorDplusToPiKPi {
       }
 
       // track-level PID selection
-      int pidTrackPos1Pion = selectorPion.statusTpcAndTof(trackPos1);
-      int pidTrackNegKaon = selectorKaon.statusTpcAndTof(trackNeg);
-      int pidTrackPos2Pion = selectorPion.statusTpcAndTof(trackPos2);
+      int pidTrackPos1Pion = selectorPion.statusTpcOrTof(trackPos1);
+      int pidTrackNegKaon = selectorKaon.statusTpcOrTof(trackNeg);
+      int pidTrackPos2Pion = selectorPion.statusTpcOrTof(trackPos2);
```

Comment:
Instead of having an hard-coded strategy, I would add a configurable as done for the D0 (see https://github.com/AliceO2Group/O2Physics/blob/f535eb32a9c5266c10857c4957b0b0a5863da757/PWGHF/TableProducer/candidateSelectorD0.cxx#L52)